src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,32 @@
+.cards {
+  --columns-count: 1;
+  display: grid;
+  column-gap: 48px;
+  row-gap: 46px;
+  grid-template-columns: repeat(var(--columns-count), 200px);
+  justify-content: center;
+
+  max-width: max-width(--columns-count, 200px, 48px);
+
+  @function max-width ($cards-amount, $card-width, $gap-width) {
+    $result: (($card-width * $cards-amount)
+    + ($gap-width * ($cards-amount - 1)));
+
+    @return $result;
+  }
```

Comment:
Your function doesn't work 😢 Try returning this `(($card-width * $cards-amount) + ($gap-width * ($cards-amount - 1)))` (without $result variable)